### DISCUSSION TITLE
Using alchemy models

### DISCUSSION THREAD

**Discussion by nbraun-wolf** [encode/orm#138] [Q&A] [answered]
Hi, I am wondering if its possible to feed sql alchemy model classes into this orm.

**Comment by aminalaee** [comment, marked as answer]
Hey, there's no direct way to do this automatically, so maybe you need to create an external tool to transform SQLAlchemy models to ORM models.

**Reply by nbraun-wolf** [reply]
OK, thank you. I thought since it's using sqlalchemy under to hood to generate queries, it's already using some kind of sqlalchemy model under the hood.

**Reply by aminalaee** [reply]
At the moment it's not possible.
And also note that you can take advantage of SQLAlchemy's features at the same time, like using SQLAlchemy filters and getting SQLAlchemy tables from ORM.
